packages/router-plugin/tests/router-generator-plugin-watcher.test.ts

```
@@ -0,0 +1,116 @@
+import { mkdir, mkdtemp, readFile, rm, writeFile } from 'node:fs/promises'
+import { tmpdir } from 'node:os'
+import path from 'node:path'
+import { afterEach, beforeEach, describe, it } from 'vitest'
+import { createServer } from 'vite'
+import type { ViteDevServer } from 'vite'
```

ESLint

[error] `vite` type import should occur after import of `../src/vite`

(import/order)